### ISSUE TITLE
Default

### ISSUE THREAD

**Issue by Realnameisjames** [OneUIProject/oneui-design#39]

**Comment by Realnameisjames** [comment]
> 

Replying to the mandated text I cannot find now, CYBRETEK and it's copyright to works not contributed but made publicly and without any choice given to accessibility, wanted there being none, but stolen from day one in my epik.co domain and fiscal.org purchase as well...

**Comment by Yanndroid** [comment]
What?
